drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -42,6 +42,7 @@ class Application::User < Types::BaseObject
     field :manage_account_url, String, null: false
 
     field :staff_assignments, HmisSchema::StaffAssignment.page_type, null: true
+    field :ce_assigned_steps, HmisSchema::CeReferralStep.page_type, null: true
```

Comment:
Another benefit of the approach proposed on https://github.com/greenriver/hmis-warehouse/pull/5419/files#r2104889700 is that this field can entirely from the application `User`  type onto `MyDashboard` type, which means that access checks become simpler because we know we're only ever resolving the data (in this case, assigned steps) for the `current_user`.

In other words-- if this field is added to User, we probably need to add `raise unless current_user == object` because we shouldn't allow users to resolved assigned steps for _other_ users.